hw_diag/tests/data/test-key-pub.gpg

```diff
@@ -0,0 +1,23 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
If you are 1000000% positive this is a test key not used anywhere else, you might as well add the private key too, in case we every want to use it for any tests in the future.

Reply:
I am sure a test key won't be used elsewhere. The test key can't compromise the security and it has no relation with the official key. So I am going to add the private key that correspond to the test public key.